github/client_repository_pullrequest.go

```diff
@@ -66,6 +66,16 @@ func (c *PullRequestClient) Create(ctx context.Context, title, branch, baseBranc
 	return newPullRequest(c.clientContext, pr), nil
 }
 
+func (c *PullRequestClient) Edit(ctx context.Context, number int, opts gitprovider.EditOptions) (gitprovider.PullRequest, error) {
```

Comment:
The docs says 
> The following fields are editable: Title, Body, State, Base.Ref and MaintainerCanModify. Base.Ref updates the base branch of the pull request.

Here it seems we only update the title. Can we add a comment stating that we only handle title updates?

Reply:
Given you *have* to provide `gitprovider.EditOptions` and that only has a `Title` field at this point, isn't that explicit enough about which fields are editable?

Reply:
Given the function name, I would still add it to avoid ambiguity.

Reply:
Will do.